src/scripts/_button-set-gallery.js

```diff
@@ -1,10 +1,12 @@
-const gallery = document.querySelector('.gallery');
-const buttons = gallery.querySelectorAll('.gallery__button');
-const photos = gallery.querySelectorAll('.gallery__photo');
+const gallery = document.querySelector('.js-gallery');
+const buttons = gallery.querySelectorAll('.js-gallery-button');
+const photos = gallery.querySelectorAll('.js-gallery-photo');
 
-[...buttons].forEach(button => {
-  button.addEventListener('click', changeGalleryPhotos);
-});
+export function addListenerForGalleryButtons() {
+  [...buttons].forEach(button => {
+    button.addEventListener('click', changeGalleryPhotos);
+  });
+};
 
 function changeGalleryPhotos({ target }) {
```

Comment:
line 21: don't rely on innerText
if somebody change text in HTML your script will stop working

better to have smth like:
```html
<button data-index="0" data-gallery-button>
<button data-index="1" data-gallery-button>

...
<div data-index="0" data-gallery-tab>
  <img ...
  <img ...
  <img ...
</div>

<div data-index="1" data-gallery-tab>
  <img ...
  <img ...
  <img ...
</div>